code/game/objects/items/cigs_lighters.dm

```diff
@@ -333,7 +333,7 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	list_reagents = list(/datum/reagent/drug/nicotine = 20, /datum/reagent/medicine/regen_jelly = 15, /datum/reagent/drug/krokodil = 4)
 
 /obj/item/clothing/mask/cigarette/nonico
-	desc = "A nicotine-free cigarette."
+	desc = "A Tobacco-free cigarette, it contains Chamomile, Lemon Balm, and Catnip."
```

Comment:
```suggestion
	desc = "A tobacco-free cigarette, it contains chamomile, lemon balm, and catnip."
```

Reply:
i think the "ingredients" are fine, they literally don't have to mean anything. just don't capitalize them